index.js

```diff
@@ -37,11 +41,30 @@ var good = {
   }
 };
 
-server.register([Vision, Inert, loutRegister, good], function(err) {
+var session = {
+  register: yar,
+  options: {
+    maxCookieSize: 0,
+    cache: {
+      expiresIn: 24 * 60 * 60 * 1000
+    },
+    cookieOptions: {
+      password: process.env.Password || 'abcdefghigklmnopqrsdasdasdadadadsdtuvwxyz123456',
```

Comment:
@Rub21 @geohacker Not sure about the context here but is it okay to have a hard-coded password?